kegg_pathways_completeness/bin/parse_hmmtable/README.md

```diff
@@ -11,13 +11,29 @@ Protein fasta file (PROTEIN_SEQS)
 wget https://ftp.ebi.ac.uk/pub/databases/metagenomics/pipeline-5.0/ref-dbs/db_kofam.hmm*
 ```
 
-## Run
+## Run 
+
+You can use hmmscan or hmmsearch
+
+### - hmmscan
 
 ```commandline
 hmmscan --noali --cut_ga --domtblout <OUTPUT FILE NAME> ${REFDB}/db_kofam/db_kofam.hmm <PROTEIN_SEQS>
  
 # parse table
-python3 make_hmmscan_table_tab_separated.py -i <OUTPUT FILE NAME> -o <output_name for tab-separated file>
+python3 make_hmmer_table_tab_separated.py -i <OUTPUT FILE NAME> -o <output_name for tab-separated file>
+
+python3 parse_hmmer_tab_separated_table.py -t hmmscan -i <output_name for tab-separated file> -f <PROTEIN_SEQS>
```

Comment:
Not for this version, but maybe in the future you can use https://pyhmmer.readthedocs.io/en/stable/examples/index.html to easier. Pyhmmer has some utilities to handle outputs (and to run hmmer from within python)

Reply:
That tool is not only for hmmer output, it is general. Hmmer run is just one of examples.

Reply:
ahh.. sorry I got it wrong.